Unified.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Nov 14 12:30:49 2021
+
+@author: DianaG
+"""
+
+import ldap
+import csv
+import pandas as pd
+import datetime
+import dateutil.relativedelta
+
+# **** Part 1/2 ****
+
+ldap_server = 'ldap://172.20.1.216:389'
```

Comment:
If you want some practice, replace these 4 costs (server, dn, user, password) with console arguments using "argparse"